cmd/kube-scheduler/app/server.go

```diff
@@ -156,6 +156,9 @@ func runCommand(cmd *cobra.Command, args []string, opts *options.Options) error
 
 // Run executes the scheduler based on the given configuration. It only return on error or when stopCh is closed.
 func Run(cc schedulerserverconfig.CompletedConfig, stopCh <-chan struct{}) error {
+
+	// To help debugging, immediately log version
+	klog.Infof("Version: %+v", version.Get())
```

Comment:
i'm -1 on this change because it prints a version on each invocation of the Run() function.
this should be at least wrapped with klog.V(1)....

Reply:
would leave the approval to the maintainers at this point.